server/base/src/main/java/org/apache/accumulo/server/metadata/ServerAmpleImpl.java

```diff
@@ -157,7 +159,8 @@ public void deleteGcCandidates(DataLevel level, Collection<String> paths) {
     } else if (level == DataLevel.METADATA || level == DataLevel.USER) {
       Range range = MetadataSchema.DeletesSection.getRange();
       if (continuePoint != null && !continuePoint.isEmpty()) {
-        String continueRow = MetadataSchema.DeletesSection.getRowPrefix() + continuePoint;
+        String continueRow = MetadataSchema.DeletesSection.getRowPrefix()
```

Comment:
When I first looked at this I was not sure this would still work as intended because of the different sort order.  Thinking through it, it still seems to work though.  This would be follow on work, but if it does not exist we probably need a test to ensure when using `continuePoint` that all data is read.  I will open an issue.

Reply:
opened #1351